clinic/clinic-user-guide-for-tiup.md

```diff
@@ -0,0 +1,315 @@
+```bash
+[root@Copy-of-VM-EE-CentOS76-v1 qiaodan]# tiup diag upload /home/qiaodan/diag-fNTnz5MGhr6
+Starting component `diag`: /root/.tiup/components/diag/v0.5.1/diag upload /home/qiaodan/diag-fNTnz5MGhr6
+Enter Username: username
+Enter Password: >>>>>>>>>>>>>>>>>>>>>>>>>>>>>><>>>>>>>>>
+Completed!
+Download URL: "https://clinic.pingcap.com:4433/diag/files?uuid=52679daa98304e43-82efa642ce241f81-8694e4a10c5736ce"
+```
+  - 上传需要提供用户名和密码，目前 Clinic 在 Beta 受邀测试使用阶段，请联系与您对接的 PingCAP 技术人员或者联系 clinic-trail@pingcap.com 获取试用账号。
+  - 上传完成后返回数据访问链接，请将 `Download URL` 中的数据访问链接发给 PingCAP 技术支持人员。目前 Clinic Server 的数据访问链接只对 PingCAP 技术支持人员开放，上传数据的外部用户暂时无法打开该链接。
+  
+
+#### 先打包后上传
+适用场景：集群是离线部署，所在的网络无法直接连接 Clinic Server ，需要先在内网打包，然后发送到网络连通的设备上进行上传。
+操作方式：
+1. 先使用 Package 命令对第二步收集的数据包进行压缩和加密，然后将压缩后的文件传到非隔离网络。
+ {{< copyable "shell-regular" >}}
+```bash
+tiup diag package <filepath>
+```
+ {{< copyable "shell-regular" >}}
+```bash
+Starting component `diag`: /root/.tiup/components/diag/v0.5.1/diag package diag-fNTnz5MGhr6
+packaged data set saved to /home/qiaodan/diag-fNTnz5MGhr6.diag
+```
+  - 打包时会同时进行加密和压缩。测试环境中 800MB 数据压缩后变为 57MB。
+  - 打包后文件将会是 xxxx.diag，是可以被正常上传的，不需要进行额外处理。
+  - 如果用户想直接转发收集的数据并不在 Clinic Server 查看，可以自行压缩转发。 `Package` 命令压缩后的数据包必须上传到 Clinic Server 后才能解密查看。
+
+2. 在可以连通 Clinic Server 的网络环境下上传压缩包：
+ {{< copyable "shell-regular" >}}
+```bash
+tiup diag upload filepath -u=username -p='password'
+```
+{{< copyable "shell-regular" >}}
+```bash
+[root@Copy-of-VM-EE-CentOS76-v1 qiaodan]# tiup diag upload /home/qiaodan/diag-fNTnz5MGhr6
+Starting component `diag`: /root/.tiup/components/diag/v0.5.1/diag upload /home/qiaodan/diag-fNTnz5MGhr6
+Enter Username: username
+Enter Password: >>>>>>>>>>>>>>>>>>>>>>>>>>>>>><>>>>>>>>>
+Completed!
+Download URL: "https://clinic.pingcap.com:4433/diag/files?uuid=52679daa98304e43-82efa642ce241f81-8694e4a10c5736ce"
+```
+  - 上传需要提供用户名和密码，目前 Clinic 在 Beta 受邀测试使用阶段，请联系与您对接的 PingCAP 技术人员或者联系 clinic-trail@pingcap.com 获取试用账号。
+  - 上传完成后返回数据访问链接，请将 `Download URL` 中的数据访问链接发给 PingCAP 技术支持人员。目前 Clinic Server 的数据访问链接只对 PingCAP 技术支持人员开放，上传数据的外部用户暂时无法打开该链接。
+
+### 可选操作：本地查看数据
+所有收集到的数据被按来源存储在以机器名和端口号命名的独立子目录中，对每个节点，其中的配置、日志等文件均按照其在真实服务器上相同的相对路径存放。
+其中：
+- 基础的系统和硬件信息在 `insight.json` 中
+- 系统 `/etc/security/limits.conf` 中的内容在 `limits.conf` 中
+- 内核参数列表在 `sysctl.conf` 中
+- 内核日志在 `dmesg.log` 中
+- 采集当时的网络连接情况在 `ss.txt` 中
+- 配置数据在每节点目录下的 `config.json` 中
+- 集群本身的元信息在 `meta.yaml` 中，此文件位于采集数据存储目录的顶层。
+- 监控数据在 `/monitor` 文件目录中，默认经过压缩，无法直接查看，如需直接查看监控指标的 JSON 文件内容，可在采集时通过 `--compress-metrics=false` 参数禁用压缩。
+
+## 使用 Clinic Diag 工具快速诊断集群
+
+Clinic Diag 工具支持对集群的健康状态进行快速的诊断，目前版本主要支持配置项内容检查，快速发现不合理的配置项。
+
+### 使用步骤
+（1）采集配置数据 
+ {{< copyable "shell-regular" >}}
+```bash
+tiup diag collect <cluster-name> --include="config" 
+```
+- 默认的采集数据会放到当前路径下。配置文件数据较小，测试环境 18 节点集群数据量小于 10KB。
+
+（2）对配置数据进行诊断
+ {{< copyable "shell-regular" >}}
+```bash
+tiup diag check <subdir-in-output-data> 
+```
+- <sudir-in-output-data>  是用 `diag collect` 命令收集数据后包含 `meta.yaml` 的目录 。
```

Comment:
```suggestion
## 使用 Clinic Diag 工具快速诊断集群

Clinic Diag 工具支持对集群的健康状态进行快速诊断。目前 Clinic XX 版本主要支持配置项内容检查，快速发现不合理的配置项。

### 使用步骤

1. 采集配置数据 

    {{< copyable "shell-regular" >}}
    
    ```bash
    tiup diag collect <cluster-name> --include="config" 
    ```

    采集数据会默认存放至当前路径下。配置文件数据较小，测试环境中的 18 节点集群数据量小于 10KB。

2. 诊断配置数据

    {{< copyable "shell-regular" >}}

    ```bash
    tiup diag check <subdir-in-output-data> 
    ```

    其中，`<sudir-in-output-data>` 是用 `diag collect` 命令存放已收集的数据的目录，其中包含 `meta.yaml` 文件 。
```